ocaml/src/tools/alba_wrappers_stubs.c

```diff
@@ -36,9 +36,10 @@ but WITHOUT ANY WARRANTY of any kind.
    would make sense.
 */
 CAMLprim value alba_get_maxrss(value unit) {
-    CAMLparam1 (unit);
+    CAMLparam1(unit);
+    CAMLlocal1(vres);
 
-    caml_release_runtime_system();
+    //caml_release_runtime_system();
```

Comment:
if getrusage is considered to be a cheap call we should probably also remove Alba_wrappers.Sys2.lwt_get_maxrss?